recipes/pyquil/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "pyquil" %}
+{% set version = "1.8.0" %}
+{% set sha256 = "08b30661e32eddd1fe609d8b55e83740443fbda76ae1b56191f2b2e816d4a4ae" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - pyquil-config-setup=pyquil.setup.pyquil_config_setup:main
+  # script: python setup.py install --single-version-externally-managed --record record.txt
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - requests >=2.4.2
+    - numpy 1.11.*
+    - matplotlib >=1.5
+    - typing >=3.6
+    - urllib3 >=1.21.1
+    - antlr-python-runtime >=4.7
+    - pytest-runner
+  run:
+    - python
+    - requests >=2.4.2
+    - numpy >=1.11
+    - matplotlib >=1.5
+    - typing >=3.6
+    - urllib3 >=1.21.1
+    - antlr-python-runtime >=4.7
+
+test:
+  imports:
+    - pyquil
+    - pyquil._parser
+    - pyquil._parser.gen2
+    - pyquil._parser.gen3
+    - pyquil.api
+    - pyquil.latex
+    - pyquil.setup
+  commands:
+  # This command configures pyquil to connect to Riggeti Forest. It prompts the
+  # user for an API key and username. It's appropriate for local testing, but
+  # not conda forge. see http://pyquil.readthedocs.io/en/latest/start.html#connecting-to-the-rigetti-forest
+    # - pyquil-config-setup --help
```

Comment:
Does this need user interaction even for `--help`? It's generally a good idea to test that installed commands run at all with something like `--help`.